app/views/users/registrations/edit.html.erb

```diff
@@ -8,34 +7,36 @@
       <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
         <%= devise_error_messages! %>
         <div class="form-group">
-        <%= f.label :email, class: 'sr-only' %>
+        <%= f.label :email, class: ' ' %>
```

Comment:
See [the above note](https://github.com/engwebUM/projecto-g1/pull/11/files#r31391587).